pkg/render/embed/initialize_keystore.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I can't say I'm a fan of this living in the operator, I don't believe we should be injecting bash commands into any container from the operator (this is more than just configuring a container to run).

If this is needed to run, then either create a separate init container to run, or add it to the elasticsearch container and put it behind a flag or env variable.